src/plone/restapi/services/aliases/add.py

```diff
@@ -83,15 +90,35 @@ def edit_for_navigation_root(self, alias):
 class AliasesRootPost(Service):
     """Creates new aliases via controlpanel"""
 
-    def reply(self):
-        data = json_body(self.request)
+    def _reply_csv(self):
+        form = self.request.form
+        if not form.get("file"):
+            raise BadRequest("No file uploaded")
```

Comment:
We should also check that the uploaded file has content-type text/csv

Reply:
@davisagli  I dont think so this is possilbe ive tried locally, In `Zope`, the `ZPublisher.HTTPRequest.FileUpload` object does not directly expose a `content_type` attribute, So this file is not having any `content_type` attribute.

Reply:
Fixed at #1821